### PR TITLE
Fix error in PipelineRun, Task, and TaskTree stories

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Fix 'invalid hook call' errors in the PipelineRun, Task, and TaskTree
stories that currently prevent them from rendering. This appears to be
caused by some unusual behaviour in Storybook relating to the default
wrapper component in the latest release.

Wrapping the affected stories in a simple decorator that just renders the
story unmodified resolves the error.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/main/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -44,6 +44,11 @@ export const parameters = {
 
 export const decorators = [
   (story, context) => (
-    <Container story={story} notes={context.parameters.notes} />
+    <Container
+      path={context.args.path}
+      route={context.args.route}
+      story={story}
+      notes={context.parameters.notes}
```
Comment (briangleeson) [inline .storybook/preview.js:51]: Nit: alphabetical order the props

**Review by briangleeson — CHANGES_REQUESTED** [review]
tiny nit, otherwise good

File: .storybook/preview.js (hunk 1 above)
Comment (AlanGreene) [reply .storybook/preview.js]: you got it 👍  done

**Comment by AlanGreene** [comment]
npm install didn't run successfully, we'll need to update the scripts to bail when this happens for quicker feedback

/test pull-tekton-dashboard-integration-tests

**Review by briangleeson — APPROVED** [review]
/lgtm

**Pull request merged**